test/utils/cryptography.js

```diff
@@ -14,7 +14,7 @@
  *
  */
 import elements from 'lisk-elements';
-import crypto from '../../src/utils/cryptography';
+import * as cryptography from '../../src/utils/cryptography';
 
 describe('crypto utils', () => {
 	describe('elements throws error', () => {
```

Comment:
Since we're just passing these through I don't think we need this suite any more.